cloud-config/run-scripts/ami-elasticsearch.sh

```diff
@@ -88,3 +101,15 @@ if [ "$ENCD_BUILD_TYPE" == 'es-nodes' ]; then
     # We could have alternativily created an app-wrapper.sh for elasticsearch
     touch "$encd_is_installed_flag"
 fi
+
```

Comment:
Is this still for Kibana?

Reply:
Yes, I will move this over to kibana branch.